templates/admin/job_template/index.html.ep

```diff
@@ -73,12 +73,16 @@
                 </span>
             </button>
         </form>
-        <h2>
-            Jobs for
-            <span id="job-group-name"><%= $group->name %></span>
-        </h2>
+        <h2>Job group <span id="job-group-name"><%= $group->name %></span></h2>
         %= include 'layouts/info'
         %= include 'admin/group/group_property_editor', group => $group, is_parent => 0
+        <h3>Job templates</h3>
```

Comment:
The two titles look weird to me:

![Screenshot from 2020-01-14 16-33-24](https://user-images.githubusercontent.com/1204189/72357698-b2035a00-36eb-11ea-8c8b-51d08f118108.png)

Also the text that now spans across the whole width, rather than staying on the left. And it feels like an excerpt from the docs, but somehow it only explains one particular concept.

Reply:
I also don't like the two consecutive headings but I think it is still clearer this way. Especially if you click on 'Edit job group properties' it makes more sense.

Reply:
Yea, that's doesn't look right to me either.

Reply:
I've changed it to adjusted the headings according to the group property editor's visibility.